gamestonk_terminal/stocks/dark_pool_shorts/yahoofinance_view.py

```diff
@@ -0,0 +1,41 @@
+__docformat__ = "numpy"
+
+import os
+from tabulate import tabulate
+from governance.helper_funcs import export_data
```

Comment:
I think here should be `gamestonk_terminal` and not `governance`. Terminal is crashing here